wrangler-service/src/main/java/co/cask/wrangler/service/explorer/FilesystemExplorer.java

```diff
@@ -126,20 +168,34 @@ public void read(HttpServiceRequest request, HttpServiceResponder responder,
     if (scope == null || scope.isEmpty()) {
       scope = WorkspaceDataset.DEFAULT_SCOPE;
     }
+    
+    final String myscope = scope;
```

Comment:
looks like you are not making any change in scope here, in that case avoid it, it will make merging https://github.com/Guavus/wrangler/pull/35 difficult.

Reply:
It was by mistake. Reverted. thanks